script.module.clouddrive.common/resources/language/Hebrew/strings.po

```diff
@@ -0,0 +1,278 @@
+# Kodi Media Center language file
```

Comment:
Your translations are using the old folder style, would be great if you can migrate to the new one. Here's an example:
https://github.com/xbmc/repo-scripts/tree/krypton/script.trumptweets/resources/language/resource.language.en_gb